battleship.py

```diff
@@ -278,3 +278,4 @@ def runSimulation(w, h):
     ## Finally, run the simulation to test it manually ##

```

Comment:
Hi Priyusha,

The variable names that are unclear which you have used in the emptyGrid are list_1 and list_2. Would you please change the inconsistent terms to meaning full names?